packages/monorepo-scripts/src/deps_versions.ts

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+import * as fs from 'fs';
+import {sync as globSync} from 'glob';
+import * as _ from 'lodash';
+
+interface Dependencies {
+    [depName: string]: string;
+}
+interface Versions {
+    [packageName: string]: string;
+}
+interface VersionsByDependency {
+    [depName: string]: Versions;
+}
+
+function log(...args: any[]) {
+    console.log(...args); // tslint:disable-line:no-console
+}
+
+function getDependencies(path: string): Dependencies {
+    const file = fs.readFileSync(path).toString();
+    const parsed = JSON.parse(file);
```

Comment:
This could throw, let's catch it and throw a more helpful error msg.

Reply:
Yeah... Although it can only throw if we have a malformed package.json.
And it will throw smth like: 'Not a valid json line:column'
What can be better than that? NPM throws the same message.

Reply:
Ok, I guess that is fine.